### PR TITLE
feat: added hover effect to card and border color to search btn.

### PR DESCRIPTION
**Pull request by JayrajSingh65**
- hey check this out

### CODE REVIEW

**Comment by JayrajSingh65** [comment]
I think this will able to merge

**Comment by CandidDeer** [comment]
@Syknapse Since this is a general change to the look of our project, please could you check this out and merge if you like it? Thank you! ✨

**Comment by CandidDeer** [comment]
> * hey this repo is showing EXCLUDED on hacktober website

@JayrajSingh65 @Syknapse Really? 😱  Can you please share the link where it says that?

**Comment by JayrajSingh65** [comment]
Screenshot 2022-10-06 122000

**Comment by CandidDeer** [comment]
> Screenshot 2022-10-06 122000

Ah, okay. I thought it was generally excluded (the entire repo) - your screenshot only shows that your PR is not yet included as it hasn't been merged yet. It hasn't been merged yet because we are waiting for review from Syk, who created the repo.

File: assets/style.css
Comment (StepanNaryshkov) [inline assets/style.css:266]: it makes sense to use css variable for it

Comment (JayrajSingh65) [reply assets/style.css:266]: What happened can u please describe it?
Because I am a beginner so it will be helpful for me I am gonna learn new things .

Comment (JayrajSingh65) [reply assets/style.css:266]: I am a beginner so can you please tell me why?

Comment (JayrajSingh65) [reply assets/style.css:266]: > it makes sense to use css variable for it
- Are you asking or telling?

Comment (StepanNaryshkov) [reply assets/style.css:266]: I am not the maintainer of this project, but it looks like all CSS file uses CSS variable, so it makes sense to follow the general style + For some colors, you used twice, it is better to not repeat it and create a variable for it.

File: assets/style.css
Comment (Syknapse) [inline assets/style.css:66]: To be honest I have to say I found the color here a bit distracting. I prefer to keep it `border: none`. But I do like the border-radius, you can keep that.

File: assets/style.css
Comment (Syknapse) [inline assets/style.css:211]: Try to use the same color as the text around it which is `var(--pink)` instead of `#ff0037`. Or go to a contrasting color. Having two close Shades of pink doesn't look very nice in my opinion.

File: assets/style.css
Comment (Syknapse) [reply assets/style.css:266]: What my colleague is trying to tell you here is that if we use the same color in more than one place in the CSS file we usually will save it to a variable. If you look right at the top of the CSS file you will see a block called `:root` and within it you see colors that are given names like `--light-grey`. These are the variables. And you can then use them in any part of the CSS file with the syntax `var(--light-grey)`. So you can add the color to the variables on top and give it a name, then use this var syntax instead of the color.

**Review by Syknapse — CHANGES_REQUESTED** [review]
Just a few changes required and we can merge it. Thank you for the contribution.

**Comment by StepanNaryshkov** [comment]
> > Screenshot 2022-10-06 122000
> 
> Ah, okay. I thought it was generally excluded (the entire repo) - your screenshot only shows that your PR is not yet included as it hasn't been merged yet. It hasn't been merged yet because we are waiting for review from Syk, who created the repo.

I have the same issue as the owner of this PR. Do not you know why hacktoberfest does not count these PRs? My 2 PRs were already merged.
image

File: assets/style.css
Comment (JayrajSingh65) [reply assets/style.css:266]: > I am not the maintainer of this project, but it looks like all CSS file uses CSS variable, so it makes sense to follow the general style + For some colors, you used twice, it is better to not repeat it and create a variable for it.

Thanks for sharing your experience 🙌

**Comment by Syknapse** [comment]
Yeah, it seems the organisation has excluded this repo from hacktoberfest, which is unfortunate for us, but also understandable under their rules. Hacktoberfest is meant to help open source projects get useful contributions and ours is only a tutorial project. We still appreciate all PRs that are improving and enhancing our project though.

**Review by JayrajSingh65 — COMMENTED** [review]
check this

**Pull request closed without merging**
